components/LeagueSurvivors/LeagueSurvivors.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { LeagueSurvivors } from './LeagueSurvivors';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+describe('LeagueSurvivors', () => {
+  it('renders correctly', () => {
```

Comment:
Renders what correctly?

Reply:
Updated verbiage included in latest commmit.